webserver/shared/routes.ts

```diff
@@ -78,4 +83,9 @@ export type EndpointTypes = {
     input: DexLastPriceRequest;
     response: DexLastPriceResponse;
   };
+  [Routes.delegationForAddress]: {
```

Comment:
You will also have to re-generate the docs for this to properly appear in the OpenAPI definition